src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -45,19 +46,29 @@ import {
 import InjectedModuleView from '~/components/PluginInjectionSite/module'
 import { toastSuccess } from '~/components/Shared/Utils'
 import withLanguage from '~/components/Util/withLanguage'
-import { getCurrentFlow, getCurrentFlowNode, RootReducer } from '~/reducers'
+import {
+  getCallerFlowsOutcomeUsage,
+  getCurrentFlow,
+  getCurrentFlowNode,
+  getReusableWorkflows,
+  RootReducer
+} from '~/reducers'
 import storage from '~/util/storage'
 import {
   defaultTransition,
   DiagramManager,
   DIAGRAM_PADDING,
   nodeTypes,
+  outcomeNodeTypes,
   Point
 } from '~/views/FlowBuilder/diagram/manager'
 import { DeletableLinkFactory } from '~/views/FlowBuilder/diagram/nodes/LinkWidget'
 import { SkillCallNodeModel, SkillCallWidgetFactory } from '~/views/FlowBuilder/diagram/nodes/SkillCallNode'
 import { StandardNodeModel, StandardWidgetFactory } from '~/views/FlowBuilder/diagram/nodes/StandardNode'
 import { textToItemId } from '~/views/FlowBuilder/diagram/nodes_v2/utils'
+import { FailureNodeModel } from '~/views/FlowBuilder/diagram/nodes_v2/FailureNode'
+import { SubWorkflowNodeModel, SubWorkflowWidgetFactory } from '~/views/FlowBuilder/diagram/nodes_v2/SubWorkflowNode'
+import { SuccessNodeModel } from '~/views/FlowBuilder/diagram/nodes_v2/SuccessNode'
```

Comment:
Success and failure nodes are part of "BlockModel" and we should also use it for the SubWorkflowNode, so these 3 lines will need to disappear